phases/is_public.go

```diff
@@ -1,9 +1,55 @@
 package phases
 
-import "fmt"
+import (
+	"bufio"
+	"os"
+	"strconv"
+	"strings"
 
-// IsPublic ...
+	"github.com/bitrise-io/go-utils/log"
+)
+
+type visibilityOption struct{
+	Name string
+	IsPublic bool
+}
+
+// IsPublic returns the whether the Bitrise project
+// should be public or not.
 func IsPublic() (bool, error) {
-	fmt.Println("SetIsPublic")
-	return false, nil
+	options := []visibilityOption{
+		visibilityOption{"Private", false},
+		visibilityOption{"Public", true},
+	}
+
+	log.Infof("SET THE PRIVACY OF THE APP")
+	log.Infof("==========================")
+	for i, opt := range options {
+		log.Printf("%d) %s", i + 1, opt.Name)
+	}
+
+	var choice int
+	for !isValid(choice, len(options)) {
```

Comment:
same as above, this check will run in each iteration even if there was any other issue which blocked to have the choice var filled